extide/gradle/src/org/netbeans/modules/gradle/GradleProjectCache.java

```diff
@@ -88,6 +88,8 @@
     private static AtomicLong timeInLoad = new AtomicLong();
     private static AtomicInteger loadedProjects = new AtomicInteger();
 
+    private static final boolean DEBUG_GRADLE_INFO_ACTION = Boolean.getBoolean("netbeans.debug.gradle.info.action"); //NOI18N
```

Comment:
Isn't this an API? If so, I'd better be documented in `arch.xml` as `<arch group="systemproperty" stability="private" .../>`